src/screens/augmentedReality/ARShowScreen.js

```diff
@@ -153,12 +160,26 @@ export const ARShowScreen = ({ navigation, route }) => {
 };
 
 const AugmentedRealityView = ({ sceneNavigator }) => {
-  const { object, isStartAnimation, setIsObjectLoading } = sceneNavigator.viroAppProps;
+  const {
+    isObjectLoading,
+    setIsObjectLoading,
+    isStartAnimationAndSound,
+    setIsStartAnimationAndSound,
+    object
+  } = sceneNavigator.viroAppProps;
 
   return (
     <ViroARScene dragType="FixedToWorld">
       <ViroAmbientLight color={'#fff'} />
 
+      {object.mp3 && !isObjectLoading && (
+        <ViroSound
+          source={{ uri: object.mp3 }}
+          paused={!isStartAnimationAndSound}
+          onFinish={() => setIsStartAnimationAndSound(false)}
```

Comment:
we have the animation set with `loop: true`, can this be a problem with sound, when we set the flag to `false` here on finish? will the sound stop while the animation runs multiple times?

Reply:
If the sound ends, the animation will stop automatically. This is because the animation state is set to `false` in the `onFinish` function of `ViroSound`. But if we want a loop then I can delete `onFinish` and set `loop: true` instead.

Reply:
äh, loop is already true:

```
animation={{
  loop: true,
```

that was why i asked that question.

Reply:
I can also make `loop: true` for `ViroSound`. That's what I meant.

Reply:
you have to decide what's working, i don't know.
animation and sound needs to be in sync all the time.

Reply:
The animation and the sound file run at the same time. 👍